source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,210 @@
+.. _csharp-delete-guide:
+
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to remove documents from your MongoDB
+collections using delete operations.
+
+Sample Data
+~~~~~~~~~~~
+
+The example in this guide use the ``restaurants`` collection
+from the ``sample_restaurants`` database. The documents in this
+collection are modeled by the following ``Restaurant`` class:
+
+.. literalinclude:: /includes/fundamentals/code-examples/crud/delete.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-model
+   :end-before: end-model
+
+This collection is from the :atlas:`sample datasets </sample-data>` provided
+by Atlas. See the :ref:`<csharp-quickstart>` to learn how to create a free MongoDB cluster
+and load this sample data.
+
+Delete Operations
+-----------------
+
+Use **delete operations** to remove data from MongoDB. Delete operations
+use the ``DeleteOne()`` method, which deletes *the first document* that
+matches your query filter, and the ``DeleteMany()`` method, which
+deletes *all documents* that match the filter.
```

Comment:
issue: This puts the relevant information further from the reader by structure.
issue: Since delete operations aren't defined (nor do they likely need defined), do we need to bold the term?

```suggestion
Use delete operations to remove documents that match a **query filter**. <define query filter>. Perform delete operations in MongoDB with the following methods:

- ``DeleteOne()`` method, which deletes *the first document* that
matches your query filter
- ``DeleteMany()`` method, which
deletes *all documents* that match a query filter
```

Reply:
migrated info from Parameters section to this section.